openfe/setup/methods/rbfe_methods.py

```diff
@@ -0,0 +1,189 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+"""Contains RBFE methods
+
+
+"""
+from __future__ import annotations
+
+from openfe.setup import LigandAtomMapping, LigandMolecule
+from openfe.setup.methods import FEMethod
+from typing import Dict, Union
+from pydantic import BaseModel, validator
+from openff.units import unit
+
+
+class ForcefieldSettings(BaseModel):
+    class Config:
+        arbitrary_types_allowed = True
+
+    # mapping of component name to forcefield path
+    forcefield: Dict[str, str]
+    nonbonded_electostatics = 'PME'
+    nonbonded_cutoff = 0.9 * unit.nanometer
+    constraints = 'HBonds'
+    rigid_water = True
+
+
+class StatePoint(BaseModel):
+    class Config:
+        arbitrary_types_allowed = True
+
+    temperature = 298.15 * unit.kelvin
+    pressure = 1 * unit.bar
+
+    # TODO: Validate that the units are correct (i.e. a pressure unit for pressure)
+    @validator('pressure', 'temperature')
+    def must_be_positive(cls, v):
+        if v <= 0:
+            raise ValueError("Must be positive")
+        return v
+
+
+# maybe this shouldn't be a pydantic class and instead our lambda protocol object
```

Comment:
I kind of like the idea of using a pydantic class to set everything in the `__init__` of our lambda protocol object, so I think we should keep it, since we can use pydantic to help validate what lamba protocols we allow (but that might become hairy/duplicating our lambda protocol object code)

Reply:
I think this is the idea, these `XSettings` objects are mostly ready to be unpacked into the `__init__` of the respective objects.  The pydantic layer gives us a validation and serialization mechanism.